app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/utils/getWidgetCriteriaOptions.js

```diff
@@ -158,7 +158,7 @@ const messages = defineMessages({
     id: 'WidgetCriteriaOption.filters_actions',
     defaultMessage: 'Update filter',
   },
-  [ACTIONS_WITH_BTS]: {
+  [ACTIONS_WITH_INTEGRATIONS]: {
     id: 'WidgetCriteriaOption.bts_actions',
```

Comment:
Please, also update this key (`integration_actions`) and corresponding translates

Reply:
Fixed.